src/invariant/test/ConservationOfLumensTests.cpp

```diff
@@ -161,13 +159,13 @@ TEST_CASE("Account balances changed without inflation",
         std::vector<LedgerEntry> entries1;
         std::generate_n(std::back_inserter(entries1), N,
                         std::bind(generateRandomAccount, 2));
-        entries1 = updateBalances(entries1, *app, gen);
+        entries1 = updateBalances(entries1, *app, gRandomEngine);
```

Comment:
Is there a reason to pass gRandomEngine in here?
If we must always use the global one, let just use it inside updateBalances as well instead of parameter gen (applies also to other functions and methods).

Reply:
Fair, fixed.